pkg/agent/agent.go

```diff
@@ -127,6 +141,7 @@ func (i *Initializer) initInterfaceStore() error {
 	}
 
 	ifaceList := make([]*interfacestore.InterfaceConfig, 0, len(ovsPorts))
+	uplinkIfName := i.nodeConfig.UplinkNetConfig.Name
```

Comment:
On Linux, is it possible to cause a nil pointer?

Reply:
i.nodeConfig.UplinkNetConfig is initialized in initNodeLocalConfig(), the Name will be "" by default. So it's ok.

Reply:
Got it.